### DISCUSSION TITLE
The first step

### DISCUSSION THREAD

**Discussion by nbboutique** [daniele77/cli#110] [Q&A]
What should I do to open two windows at the same time (for example Client and Server)? I am a totally new user and I don't know how to use it maybe you can help me?

**Comment by daniele77** [comment]
You have to open two consoles.
On windows:
https://www.howtogeek.com/235101/10-ways-to-open-the-command-prompt-in-windows-10/
On Linux:
https://www.lifewire.com/ways-to-open-a-terminal-console-window-using-ubuntu-4075024

**Reply by nbboutique** [reply]
Sorry, but I don`t understand how to use cli package. How can I do a cisco style interface in my cpp project? Where can I find a tutorial? ( I have installed the package)

**Reply by nbboutique** [reply]
![image](https://user-images.githubusercontent.com/46777706/114234263-2d06ff00-9987-11eb-948d-0d873f82593d.png)
I want an appearance smth like this. What commands should I use?

**Reply by daniele77** [reply]
Ok, in the `examples` directory you can find several C++ example applications.
You should compile i.e. the example `complete` or `pluginmanager` (to compile them you can use cmake or make or Visual Studio).
Then, you can launch the example from a console and... you're done!
